homework1

```diff
@@ -0,0 +1,367 @@
+import re
+import sys
+import glob
+from time import localtime
+import unittest
+from itertools import groupby
+import csv
+import itertools
+import xml.etree.ElementTree as etree
+import random
+
+# 1 line: Output
```

Comment:
It looks much more clear if you task description stands closely to task and 
separated from other tasks with two blank lines: 
```python
   pass  # last line of previous task
    
 
   # next task docstring
   pass  # my next task first line  
```